src/index.html

```diff
@@ -5,9 +5,50 @@
     <meta name="viewport"
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <title>Product cards</title>
-    <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
+
+    <link rel="stylesheet" href="./styles/index.scss">
   </head>
   <body>
-    <h1>Product cards</h1>
+    <article class="product-card" data-qa="card">
+      <img
+        src="./images/imac.jpeg"
+        alt="Product Image"
+        class="product-card__img"
+      >
```

Comment:
It's better to use a product name as "alt".
13. [SEO] - alt atribute should describe the image if the image contains information (better description you have - better for you :))
![image](https://github.com/mate-academy/layout_product-cards/assets/63243136/a59beff1-5138-4a19-9545-ea0b102eb2ac)